client/state/partner-portal/selectors/index.js

```diff
@@ -1,4 +1,8 @@
 /**
  * Internal dependencies
  */
+import getCurrentRoute from 'calypso/state/selectors/get-current-route';
+
 export * from 'calypso/state/partner-portal/selectors/partner';
+
+export const isPartnerPortal = ( state ) => /^\/partner-portal(?:\/[^/]*)?/.test( getCurrentRoute( state ) );
```

Comment:
Kind of a nitpick but can we use `startsWith` here instead of the RegEx?

Reply:
Fixed 👍